hbase-server/src/main/resources/hbase-webapps/master/table.jsp

```diff
@@ -1089,9 +1089,18 @@
 </table>
 
 <% }
-} catch(Exception ex) {
+} catch(Exception ex) { %>
+  Unknown Issue with Regions
+  <div onclick="document.getElementById('closeRegion').style.display='block';document.getElementById('openRegion').style.display='none';">
+    <a id="openRegion" style="cursor:pointer;"> Show StackTrace</a>
```

Comment:
`openRegion` and `closeRegion` ids could be renamed to `openStackTrace` and `closeStackTrace` respectively?

Reply:
Done.